src/components/Comments.jsx

```diff
@@ -0,0 +1,41 @@
+import { getArticleComments } from "../../utils/utils"
+import { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import CommentCard from "./CommentCard"
+
+function Comments (){
+    const [comments, setComments] = useState([])
+    const [loading, setLoading] = useState(true)
+    const [error, setError] = useState(null)
+    const {article_id} = useParams()
+
+
+    useEffect(()=>{
+        getArticleComments(article_id).then((response)=>{
+            setLoading(false)
+            setComments(response.data.comments)
+        }).catch((error)=>{
+            setLoading(false)
+            setError('Error fetching comments')
```

Comment:
This is may be a problem with your backend more than anything, but an article having no comments shouldn't trigger an error, rather it should just return an empty array.
So for example article with id 37 has no comments, and it should just have an empty comment section.
Since this is a backend problem, I wouldn't try and fix it today - but circle back around to it another time